src/components/CheckboxInput/CheckboxInput.module.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../styles/variables/tokens.scss';
```

Comment:
This import shouldn't be necessary since variables are applied to the :root. Commenting out this import has no impact on the component.